app/models/user.rb

```diff
@@ -44,6 +45,7 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :admin, inclusion: { in: [true, false] }
+  validates :epr_uuid, uniqueness: { case_sensitive: true }, allow_blank: true # rubocop:disable Rails/UniqueValidationWithoutIndex
```

Comment:
Rather than disabling this cop, add a unique index for the column, for reasons explained here:
https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsuniquevalidationwithoutindex